packages/react/lib/ui/useThreeDSecure.ts

```diff
@@ -23,37 +23,56 @@ export function useThreeDSecure(opts?: UseThreeDSecureOptions) {
 
   const start = useCallback(
     (session: string, callbacks?: UseThreeDSecureCallbacks) => {
-      if (instance.current) return;
+      if (instance.current) {
+        instance.current.unmount();
+        instance.current = null;
+      }
```

Comment:
just to clarify — successive calls to start a 3ds session will unmount and remount with new sessions?

Reply:
Effectively yeah because the session is most likely in a failed or error state if the user feels a need to submit again. However, this does open up to multiple sessions being created is someone clicks rapidly, so we'll need to do something smarter